feat: circle icons and dropdown adjustment

alfonsobries opened the pull request:
<!--
Thanks for your interest in the project. Bugs filed and PRs submitted are appreciated!

Please make sure you're familiar with and follow the instructions in the [contributing guidelines](https://ark.dev/docs/program-incentives/guidelines/contributing).

Please fill out the information below to expedite the review and (hopefully) merge of your pull request!
-->

## Summary
Related to: https://app.clickup.com/t/qth0ru
Dependant of:  https://github.com/ArkEcosystem/nodem/pull/142

- Icons needed for server log detail statues
- New property for changing the origin class in the dropdown (used on the table filters)


## Checklist

<!-- Have you done all of these things (where applicable)?  -->

-   [x] I checked my UI changes against the design and there are no notable differences
-   [x] I checked my UI changes for any responsiveness issues
-   [x] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I provided a screenshot of my changes to the component _(if applicable)_
-   [ ] I regenerated the `icons.html` file and checked if my newly added icon is shown correctly _(if necessary)_
-   [ ] I added an explanation on how to use the component to the readme _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

<!-- Feel free to add additional comments. -->

leMaur commented on resources/views/dropdown.blade.php:
```diff
@@ -4,6 +4,7 @@
     'buttonClassExpanded'    => 'text-theme-primary-500',
     'buttonClass'            => 'text-theme-secondary-400 hover:text-theme-primary-500',
     'dropdownClasses'        => 'w-40',
+    'dropdownOriginClass'    => 'origin-top-right',
```
🤔 Where is `dropdownOriginClass` used?

alfonsobries replied:
@leMaur its used here ArkEcosystem/nodem#142

alfonsobries commented:
@leMaur forget to add the link https://github.com/ArkEcosystem/nodem/pull/142 (the branch is not ready)

ItsANameToo commented on resources/assets/icons/status/debug.svg:
```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="icon" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 20 20" xml:space="preserve"><g transform="translate(1.5 1.5)" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10"><circle cx="8.5" cy="8.5" r="9"/><path d="M8.5 8.5V5.2m0 3.3l3.9 3.9"/></g></svg>
```
identical to 
![image](https://user-images.githubusercontent.com/35610748/128482241-deb9434e-cb65-47dd-a792-65107ead83d1.png)

alfonsobries replied:
@ItsANameToo its not identical, this one has wider lines
Waiting restart
![image](https://user-images.githubusercontent.com/17262776/128522810-3e012569-7b43-4ede-b347-ea4c64cc756b.png)
Debug:
![image](https://user-images.githubusercontent.com/17262776/128522842-f5b8560c-cf00-47c2-9961-f9a80562557f.png)

ItsANameToo replied:
can you ask oleg which one to use, they have identical designs

alexbarnsley replied:
FYI I asked Oleg about this

ItsANameToo commented on resources/assets/icons/status/error.svg:
```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="icon" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 20 20" xml:space="preserve"><g fill="none" stroke="currentColor" stroke-width="2"><circle stroke-miterlimit="10" cx="10" cy="10" r="9"/><path stroke-linecap="round" stroke-linejoin="round" d="M7 13l6-6m0 6L7 7"/></g></svg>
```
identical to 
![image](https://user-images.githubusercontent.com/35610748/128482306-19f5a241-0fe6-450e-b42d-f90a39865799.png)

ItsANameToo commented on resources/assets/icons/status/fatal.svg:
```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="icon" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 20 20" xml:space="preserve"><path d="M10 19c5 0 9-4 9-9s-4-9-9-9-9 4-9 9h0c0 5 4 9 9 9zm-6.4-2.6L16.4 3.6" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/></svg>
```
identical to 
![image](https://user-images.githubusercontent.com/35610748/128482335-0e93c994-3a19-474c-9f27-8d24bac5213d.png)

alfonsobries replied:
@ItsANameToo this one is not equal, the blocked one has wider lines

alfonsobries replied:
blocked:
![image](https://user-images.githubusercontent.com/17262776/128522872-cd6d12ec-8262-4a13-ac54-0c86f1aea06b.png)
Fatal:
![image](https://user-images.githubusercontent.com/17262776/128522901-840d223a-389d-4156-9a43-9e316bd99364.png)

ItsANameToo replied:
then it's up to oleg to define which one to use, they look identical otherwise 🤔

alfonsobries replied:
@ItsANameToo the one that I added is the only one that looks good, the one for `notifications/blocked` looks weird (same for the clock icon below)

![image](https://user-images.githubusercontent.com/17262776/128523323-ac8b06d9-9d7b-417d-87c1-73a8a518dc9a.png)

alexbarnsley replied:
FYI I asked Oleg about which one to use

ItsANameToo commented on resources/assets/icons/status/info.svg:
```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="icon" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 20 20" xml:space="preserve"><g fill="none" stroke="currentColor" stroke-width="2"><circle stroke-miterlimit="10" cx="10" cy="10" r="9"/><path stroke-linecap="round" stroke-miterlimit="10" d="M10 9.2v5"/><path stroke-linecap="round" stroke-linejoin="round" d="M10 5.8h0"/></g></svg>
```
identical to
![image](https://user-images.githubusercontent.com/35610748/128482376-a1133217-a83b-485b-9b28-d52581714c16.png)

ItsANameToo commented on resources/assets/icons/status/trace.svg:
```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="icon" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 20 20" xml:space="preserve"><circle fill="none" stroke="currentColor" stroke-width="2" stroke-miterlimit="10" cx="10" cy="10" r="9"/><path fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" d="M9.8 13.5c2.1 0 3.8-1.7 3.8-3.8 0-2.1-1.7-3.8-3.8-3.8C7.7 5.9 6 7.6 6  ...
```
I guess this one is the only new one 🤔

ItsANameToo commented on resources/assets/icons/status/warning.svg:
```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="icon" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 20 20" xml:space="preserve"><g fill="none" stroke="currentColor" stroke-width="2"><circle stroke-miterlimit="10" cx="10" cy="10" r="9"/><path stroke-linecap="round" stroke-miterlimit="10" d="M10 10.5v-5"/><path stroke-linecap="round" stroke-linejoin="round" d="M10 14h0"/></g></svg>
```
identical to 
![image](https://user-images.githubusercontent.com/35610748/128482531-db42eef0-0274-4f06-896b-e85402c653c8.png)

ItsANameToo reviewed (changes requested):
preferably:
- do not add new icons that are duplicates
- rename the toast/notification ones to be more generic, and update the respective components to make use of those so we can keep it all consistent by making them use the same icon

alfonsobries commented:
@ItsANameToo other than the two icons I commented merged the toast icons with the new ones added here inside the `status` folder and updated the references for the toasts

alexbarnsley commented on resources/assets/icons/status/trace.svg:
```diff
@@ -1 +1 @@
-<svg version="1.1" id="icon" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 20 20" xml:space="preserve"><circle fill="none" stroke="currentColor" stroke-width="2" stroke-miterlimit="10" cx="10" cy="10" r="9"/><path fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" d="M7 13l6-6m0 6L7 7"/></svg>
```
I'd rename this to "search"

alexbarnsley commented on resources/views/dropdown.blade.php:
```diff
@@ -68,7 +69,7 @@ class="flex items-center focus:outline-none dropdown-button transition-default {
         x-transition:leave-start="transform opacity-100 scale-100"
         x-transition:leave-end="transform opacity-0 scale-95"
-        class="origin-top-right absolute right-0 mt-2 z-10 dropdown {{ $dropdownClasses }} {{ $fullScreen ? 'w-screen -mx-8 md:w-auto md:mx-0' : '' }}"
+        class=" absolute right-0 mt-2 z-10 dropdown {{ $dropdownClasses }} {{ $fullScreen ? 'w-screen -mx-8 md:w-auto md:mx-0' : '' }}"
```
trim the space at the start

alfonsobries commented:
@ItsANameToo as @alexbarnsley  proposed I moved the icons to a `circle` folder and renamed the icons to something more general:

- search.svg
- checkmark.svg
- clock.svg
- cross.svg
- exclamation-mark.svg
- forbidden.svg
- info.svg
- question-mark.svg


Also created a new PR on nodem to handle these new names: https://github.com/ArkEcosystem/nodem/pull/147

alfonsobries commented on resources/views/toast.blade.php:
```diff
@@ -7,12 +7,12 @@
+        'danger' => 'status.error',
+        'success' => 'status.success',
+        'info' => 'status.info',
+        'hint' => 'status.hint',
```
@alexbarnsley like the idea of the circle folder

ItsANameToo commented:
@alfonsobries moved the `circle/clock` icon to just `clock` as it's not a clock inside a circle
